### PR TITLE
add link to rbac index from implementation

### PR DESCRIPTION
**Pull request by Joel-Ando**
Corrected typos and added a link

### CODE REVIEW

File: docs/source/rbac/tech-implementation.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Prior to 3.0, tokens stored _roles_,
 which meant their scopes were resolved on each request.
 :::
 
-API tokens grant access to JupyterHub's APIs. The Role Based Access Control (RBAC) framework allows for the requesting of tokens with specific permissions.
+API tokens grant access to JupyterHub's APIs. The [RBAC framework](https://auth0.com/docs/manage-users/access-control/rbac) allows for requesting tokens with specific permissions.
```
Comment (minrk) [inline docs/source/rbac/tech-implementation.md:46]: This is not the right link for JupyterHub RBAC.

File: docs/source/rbac/tech-implementation.md [hunk 2]
```diff
@@ -79,11 +79,11 @@ With the RBAC framework, each authenticated JupyterHub API request is guarded by
 When an API request is made, the requesting API token's scopes are again intersected with its owner's (yellow box in {ref}`Figure 2 <api-request-chart>`) to ensure that the token does not grant more permissions than its owner has at the request time (e.g., due to changing/losing roles).
 If the owner's roles do not include some scopes of the token, only the _intersection_ of the token's and owner's scopes will be used. For example, using a token with scope `users` whose owner's role scope is `read:users:name` will result in only the `read:users:name` scope being passed on. In the case of no _intersection_, an empty set of scopes will be used.
 
-The passed scopes are compared to the scopes required to access the API as follows:
```
Comment (minrk) [inline docs/source/rbac/tech-implementation.md:82]: passed is right, not parsed (here and below)

**Review by minrk — COMMENTED** [review]
Thanks for your contribution! I’ve left some suggestions inline for you to incorporate. You can update this PR with the suggestions and we will come back to review again.

File: docs/source/rbac/tech-implementation.md (hunk 1 above)
Comment (Joel-Ando) [reply docs/source/rbac/tech-implementation.md:46]: ok Sir @minrk

Comment (Joel-Ando) [reply docs/source/rbac/tech-implementation.md]: Sir @minrk i did the necessary changes as requested.

File: docs/source/rbac/tech-implementation.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Prior to 3.0, tokens stored _roles_,
 which meant their scopes were resolved on each request.
 :::
 
-API tokens grant access to JupyterHub's APIs. The Role Based Access Control (RBAC) framework allows for the requesting of tokens with specific permissions.
+API tokens grant access to JupyterHub's APIs. The [RBAC framework](https://jupyterhub.readthedocs.io/en/stable/rbac/index.html) allows for requesting tokens with specific permissions.
```
Comment (minrk) [inline docs/source/rbac/tech-implementation.md:46]: Sorry, I'm afraid this is still not the right link. It should be an internal link, e.g. `./index.md`

File: docs/source/rbac/tech-implementation.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ Prior to 3.0, tokens stored _roles_,
 which meant their scopes were resolved on each request.
 :::
 
-API tokens grant access to JupyterHub's APIs. The Role Based Access Control (RBAC) framework allows for the requesting of tokens with specific permissions.
+API tokens grant access to JupyterHub's APIs. The [RBAC framework](https://jupyterhub.readthedocs.io/en/stable/rbac/upgrade.html) allows for requesting tokens with specific permissions.
```
Comment (minrk) [inline docs/source/rbac/tech-implementation.md:46]: This should be a local reference. You can use the reference `(./index.md)`.

Comment (Joel-Ando) [reply docs/source/rbac/tech-implementation.md:46]: i have messed up again sir please give me time please

Comment (Joel-Ando) [reply docs/source/rbac/tech-implementation.md:46]: sir @minrk  this should be the local reference  https://github.com/jupyterhub/jupyterhub/blob/main/docs/source/rbac/index.md i guess?

File: docs/source/rbac/tech-implementation.md [hunk 5]
```diff
@@ -43,7 +43,7 @@ Prior to 3.0, tokens stored _roles_,
 which meant their scopes were resolved on each request.
 :::
 
-API tokens grant access to JupyterHub's APIs. The Role Based Access Control (RBAC) framework allows for the requesting of tokens with specific permissions.
+API tokens grant access to JupyterHub's APIs. The [RBAC framework](https://github.com/jupyterhub/jupyterhub/blob/main/docs/source/rbac/index.md) allows for requesting tokens with specific permissions.
```
Comment (minrk) [inline docs/source/rbac/tech-implementation.md:46]: As I suggested before, this should be a local reference (i.e. a relative path, or a myst reference target). I suggested a value that would work: ./index.md

Comment (Joel-Ando) [reply docs/source/rbac/tech-implementation.md:46]: > As I suggested before, this should be a local reference (i.e. a relative path, or a myst reference target). I suggested a value that would work: ./index.md

Capture12
 sir @minrk is above correct?

Comment (minrk) [reply docs/source/rbac/tech-implementation.md:46]: yes

**Pull request merged**